scripts/rnnlm/train_rnnlm.sh

```diff
@@ -219,8 +222,11 @@ if [ $stage -le $num_iters ]; then
   echo "$0: best iteration (out of $num_iters) was $best_iter, linking it to final iteration."
   ln -sf $embedding_type.$best_iter.mat $dir/$embedding_type.final.mat
   ln -sf $best_iter.raw $dir/final.raw
+  ln -sf $best_iter.raw $dir/rnnlm  # to make it consistent with other RNNLMs
 fi
 
+touch $dir/unk.probs
```

Comment:
once we modify this setup to have its own rescoring scripts, unk.probs may no longer be needed.   but I may merge this as-is for now.